spec/support/test_application/app/controllers/application_controller.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+
+  # def default_url_options
```

Comment:
If this is set, you need to assert that `www.example.com` appears in the output before and after the ticks, and if you make that change, the tests still pass.